docs/application/web/guides/multimedia/media-controller.md

```diff
@@ -521,6 +531,166 @@ To manage the media controller playlist in your application, you must handle req
+    console.log(JSON.stringify(ability));
+  },
+  ondisplayrotationabilitychanged: function(server, ability)
+  {
+    console.log("display rotation ability changed, server name: " + server.name + ", ability: ");
+    console.log(JSON.stringify(ability));
+  },
+  ondisplaymodeabilitychanged: function(server, ability)
+  {
+    console.log("displayMode ability changed, server name: " + server.name + ", abilities: ");
+    console.log(JSON.stringify(ability));
+  },
+  onsimpleabilitychanged: function(server, type, ability)
+  {
+    console.log(type + " ability changed, server name: " + server.name + ", ability: " + ability);
+  }
+};
+
+var watchId = client.addAbilityChangeListener(listener);
+
+/* Server-side code */
+var server = tizen.mediacontroller.createServer();
+server.abilities.playback.play = "YES";
+server.abilities.playback.saveAbilities();
+server.abilities.shuffle = "NO";
+server.abilities.repeat = "YES";
+```
+
+You will receive information about ability changes of every active media controller server. To receive information only from selected servers, calling function `subscribe()` is required.
+
+```
+var client = tizen.mediacontroller.getClient();
+var info = client.getLatestServerInfo();
+info.abilities.subscribe();
+```
+
+After first use of `subscribe()` you will stop receiving changes from not subscribed servers.
+
+To stop monitoring specific server use analogical `unsubscribe()` method. In case when no server is subscribed, notifications from all servers will be fired.
+
+
+## Server features
+
+Media controller API provides methods to change and monitor server features:
+- Subtitles
+- Mode360
+- DisplayMode
+- DisplayRotation
+
+From server side you can monitor client requests of its features by using correspondant change request listener.
+
+```
+var watcherId = 0;  /* Watcher identifier. */
+var mcClient = tizen.mediacontroller.getClient();
+var mcServerInfo = mcClient.getLatestServerInfo();
+
+/* Registers to be notified when display rotation changes. */
+watcherId = mcServerInfo.displayRotation.addDisplayRotationChangeListener(function(rotation)
+{
+  console.log(mcServerInfo.name + " server display rotation changed to " + rotation);
+});
+
+/* Cancels the watch operation. */
+mcServerInfo.displayRotation.removeDisplayRotationChangeListener(watcherId);
+```
+
+From client side you can send request of changin specific feature to server by using `sendRequest()` method. Request can be only sent after [enabling specific ability](#server-abilities) on server side.
+
+```
+var mcClient = tizen.mediacontroller.getClient();
+var mcServerInfo = mcClient.getLatestServerInfo();
+
+var rotation = "ROTATION_180";
+mcServerInfo.displayRotation.sendRequest(rotation, function(data, code)
+{
+  console.log("Server replied with return data: " + JSON.stringify(data) + " and code: " + code);
+});
+```
+
+Server can reply to your request by using `RequestReply()` method:
```

Comment:
...can reply to clients requests...

RequestReply is an object

Reply:
Done.